.circleci/config.yml

```diff
@@ -365,7 +365,12 @@ commands:
           environment:
             # Use the settings from the "ci" profile in nextest configuration.
             NEXTEST_PROFILE: ci
-          command: cargo xtask test --workspace --locked --features ci,snapshot
+          command: |
+            cargo xtask test --workspace --locked --features ci,snapshot
+
+            # run just the allowed_features integration tests with a dummy JWKS endpoint
+            APOLLO_TEST_INTERNAL_UPLINK_JWKS=true cargo xtask test --workspace --locked --features ci,snapshot,test-jwks integration_tests
```

Comment:
tell me if i'm missing something, but it should be possible to do everything in a single test run by configuring the env var inside the tests that need it